spec/controllers/api/v1/open_library_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe Api::V1::OpenLibraryController do
+  describe 'GET #show' do
+    context 'When fetching the book info' do
+      include_context 'Authenticated User'
+      @json_res = File.read('./spec/support/fixtures/show_open_library_controller_success.json')
+
+      before do
+        stubbed_service = instance_double(OpenLibrary)
+        allow(stubbed_service).to receive(:books).and_return(@json_res)
+        allow(OpenLibrary).to receive(:new).and_return(stubbed_service)
+        get :show, params: { isbn: '0451526538' }
+      end
+
+      it 'response with book info json' do
+        expect(response.body) == @json_res
```

Comment:
Use .to eq(expected) instance of ==

Reply:
had an error with the variable `@json_res`, solved it and chande the matcher